packages/calypso-e2e/src/lib/pages/site-settings-page.ts

```diff
@@ -18,37 +17,35 @@ export class SiteSettingsPage {
 	}
 
 	/**
-	 * Visits the `/sites/settings/$tab` endpoint.
+	 * Visits the `/sites/$siteSlug/settings` page and optionally drill into a sub-page.
 	 *
 	 * @param {string} siteSlug Site URL.
-	 * @param {string} tab      Settings tab.
 	 */
-	async visit( siteSlug: string, tab: string = 'site' ): Promise< void > {
-		// Patch for redirectToHostingDashboardBackportIfEnabled bug that doesn't account for tabs.
-		await this.page.goto( getCalypsoURL( `sites/settings/v2/${ siteSlug }/${ tab }` ), {
+	async visit( siteSlug: string, subPathSlug?: string ): Promise< void > {
```

Comment:
This change finally fixes the feedback that was raised with us here
p1750692623153899-slack-CRWCHQGUB

The site settings page no longer has "tabs". You can now jump directly to the settings sub page, which is sort of the new version of tabs